include/h2o/httpclient.h

```diff
@@ -155,6 +155,7 @@ typedef struct st_h2o_httpclient_ctx_t {
     size_t max_buffer_size;
     unsigned tunnel_enabled : 1;
     unsigned force_cleartext_http2 : 1;
+    unsigned forward_close_connection : 1;
```

Comment:
I'm not sure if this is the correct place to add a flag that is not used by httpclient.

Considering that this flag is used by the proxy handler, maybe we can add the flag to `h2o_req_overrides_t` the same way like `use_proxy_protocol`.

Reply:
It works well that way, done in a1d5cf0, thank you!